libraries/AP_Mount/AP_Mount_Siyi.cpp

```diff
@@ -68,6 +69,15 @@ void AP_Mount_Siyi::update()
         } else {
             request_configuration();
         }
+
+        // send UTC time to the camera
+        if (_got_hardware_id && sent_time_count < 5) {
```

Comment:
I don't think we need the _got_hardware_id here.. it's already checked above and has a return after it.

Reply:
it's an existing problem but we could also remove the "} else {" the precedes the "request_configuration".  The only reason to do this is that it makes it more clear that the request_configuration() and set_time are run together.